addons/pos_customer_display/static/src/pos_order.js

```diff
@@ -0,0 +1,26 @@
+import { patch } from "@web/core/utils/patch";
+import { PosOrder } from "@point_of_sale/app/models/pos_order";
+
+patch(PosOrder.prototype, {
+    getCustomerDisplayData() {
+        const temp = this.getSortedOrderlines().map((l) => ({
+            ...l.getDisplayData(),
+            isRefund: l.refunded_orderline_id ? true : false,
+            imageSrc: `/web/image/product.product/${l.product_id.id}/image_128`,
+        }));
+        const refund = temp.filter((x) => x.isRefund);
+        const order = temp.filter((x) => !x.isRefund);
+        return {
+            ...super.getCustomerDisplayData(),
+            partner: this.partner_id
+                ? {
+                      id: this.partner_id.id,
+                      name: this.partner_id.name,
+                  }
+                : null,
+            amount_per_guest: this.amountPerGuest(),
+            refundLines: refund,
+            lines: order,
+        };
+    },
+});
```

Comment:
```suggestion
        displayData = super.getCustomerDisplayData(),
        temp_lines = displayData.lines.map((l) => ({
            ...l,
            isRefund: l.refunded_orderline_id ? true : false, // Add isRefund property
        }));
        
        return {
            ...displayData,
            amount_per_guest: this.amountPerGuest(),
            refundLines: temp_lines.filter((x) => x.isRefund),
            lines: temp_lines.filter((x) => !x.isRefund),
        };
    },
});
```
we don't need to rerun `getSortedOrderlines` to get lines we will already have them from super use them and update as expected.